tools/run_tests/artifacts/build_artifact_python.bat

```diff
@@ -38,8 +38,9 @@ set GRPC_PYTHON_BUILD_WITH_CYTHON=1
 
 @rem Multiple builds are running simultaneously, so to avoid distutils
 @rem file collisions, we build everything in a tmp directory
-if not exist "artifacts" mkdir "artifacts"
-set ARTIFACT_DIR=%cd%\artifacts
+@rem TODO(jtattermusch): it doesn't look like builds are actually running in parallel in the same dir
```

Comment:
Because the new ```create_jobspec()``` uses ```run_in_workspace.sh```, this might night be needed anymore.
Same with below, its either run in docker or a separate workspace, either way there shouldn't be overlap.

I'm fine leaving this as is though to get things submitted.